conanfile.py

```diff
@@ -52,9 +54,21 @@ def source(self):
         extrated_dir = self.name + "-" + self.version
         os.rename(extrated_dir, self._source_subfolder)
 
+    def configure(self):
+        if(self.options.xft == "auto"):
+            self.options.xft = (self.settings.os == 'Linux' or self.settings.os == 'FreeBSD')
+        if(self.options.fontconfig == "auto"):
+            self.options.fontconfig = (self.settings.os != 'Windows' and self.settings.os == 'Macos')
```

Comment:
`self.settings.os == 'Macos` implies `self.settings.os != 'Windows'`. there is probably a typo here